src/fr/composants/champ-de-date/design.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Champ de date
+layout: 'layouts/component-documentation.njk'
+translationKey: 'dateinputDesign'
+tags: ['dateinputFR', 'design']
+date: "git Last Modified"
+---
+
+## Structure de la Champ de date
```

Comment:
Waiting from design + translation review because this portion is different from the document